components/ButtonBar/examples/ButtonBarIconExample.m

```diff
@@ -90,10 +90,6 @@ + (BOOL)catalogIsPrimaryDemo {
   return NO;
 }
 
-+ (BOOL)catalogIsPresentable {
```

Comment:
Commented in Pivotal, but I think ButtonBars are sufficiently useful on their own.  Think about how Google Docs provides text alignment button bars within their toolbars.

Reply:
Spoke offline. Clarified that ButtonBars are not currently intended to be used as a standalone component.